ext/riverlea/core/css/components/_form.css

```diff
@@ -407,7 +415,7 @@ input.crm-form-checkbox) + label {
   width: auto;
   max-width: 100%;
 }
-.crm-container .select2-container {
+.crm-container .select2-container:not(.four,.six,.eight,.twelve,.medium) {
```

Comment:
Tangential: So this formatting makes sense -- but the civilint/phpcs ruleset rejects it. It wants something goofy that makes it hard to read.  🫤 

Split out: https://github.com/civicrm/coder/pull/21